.github/workflows/verify.yml

```diff
@@ -0,0 +1,29 @@
+name: Frontend Verify
+on:
+  push:
+    branches:
+      - front-end
+  pull_request:
+    branches:
+      - front-end
+jobs:
+  build:
+    name: Verify
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2-beta
+        with:
+          node-version: '12'
+      - name: Yarn Install
+        uses: Borales/actions-yarn@v2.3.0
```

Comment:
The official node action contains `yarn`: `actions/setup-node`. Can you replace it?

Reply:
@rodrigo-brito Yeah that makes sense I've made that change, let me know if there is anything else you feel needs changing.